test/blocks/media/media.test.js

```diff
@@ -54,6 +58,10 @@ describe('media', () => {
       const links = medias[4].querySelectorAll('h3.heading-xs ~ p.subcopy-link > a');
       expect(links.length).to.greaterThanOrEqual(2);
     });
+    it('does not have cta container around mid-body action area', () => {
+      const actionArea = medias[4].querySelector('.action-area');
+      expect(actionArea.parentElement.className.includes('cta-container')).to.be.false;
```

Comment:
Same here. No need for caching.
```suggestion
      expect(medias[4].querySelector('.action-area').parentElement.className.includes('cta-container')).to.be.false;
```